### DISCUSSION TITLE
View of the duck

### DISCUSSION THREAD

**Discussion by pyskaa0506** [Quit975/DuckyGame#66] [Polls]
Quick preview how it could look like:
https://cdn.discordapp.com/attachments/885112955077816351/982746708733751376/unknown.png

**Comment by AgeTheGeek** [comment]
In my opinion 3/4 option seems like a great decision for showing off duck's character design:
- 3/4 view can help to show duck from the most appealing angle- we can show it as a set of multiple shapes, as an opposite for the top view, where her design could me mostly described as a single oval shape (3/4 gives us more creative space)
- 3/4 would add more depth to our game. BUT(!) it rises the question: do we want our game to have this depth?  I think it has to be further disscused as in the whole concept of our game art style.
However, If we don't need this depth, I think that the side view would be a great solution as well :)

**Comment by Quit975** [comment]
Please remember that the game will not be isometric. So any view you go for needs to work well with the current "plane' of movement

**Reply by Quit975** [reply]
Oh, and! 3/4 will be much more challenging, both technically, and artistically. It will need a separate set of sprites for every of the 8 directions it can go. Not saying that it's out of the question, just want to make sure that everyone is aware of the increased cost of development.

**Reply by pyskaa0506** [reply]
Technically it would only be 5 directions, because we could mirror some of them
But yeah, it's still much more

**Comment by Quit975** [comment]
I would personally go for a mix between side and 3/4. Meaning that the player is visible from the kinda-angle, but does not rotate when changing directions

**Reply by pyskaa0506** [reply]
That sounds really good to me. Let's see what others think about that idea

**Comment by NNathaneel** [comment]
Personally I'm more into side-view, but I can see how the 3/4 would work, it'd require some testing tho, because drawing this kind of thing is quite easy to screw up in 2D (requires more knowledge on the look of the drawn subject than simple side-viewed character). Also I don't know if that wouldn't affect the smoothness of the look of the game it the character couldn't easily rotate and just switched sides like in the side view. As I said the design of the duck must be tested under such circumstance to see if it'd work. When I'll find some time for the quick sketches I might come up with something.

**Comment by NNathaneel** [comment]
Top view is out of question for me. I've never liked it haha
